Patterns/Chinese/Chinese-DateTime.yaml

```diff
@@ -13,7 +13,7 @@ MonthNumRegex: !simpleRegex
 TwoNumYear: '50'
 YearNumRegex: !simpleRegex
   def: (?<year>((1[5-9]|20)\d{2})|2100)
-YearRegex: !simpleRegex
+SimpleYearRegex: !simpleRegex
```

Comment:
Why do you have this rename and other changes in Chinese and not in Japanese? Don't they follow the same design?